utils/utils.go

```diff
@@ -1058,26 +1057,36 @@ func ForkVersionAtEpoch(epoch uint64) *types.ForkVersion {
 	}
 }
 
-func LogError(infoIdentifier string, err error) *logrus.Entry {
-	var fileIdentifier, lineIdentifier, functionIdentifier string
+func LogFatal(err error, errorMsg interface{}, callerSkip int, infoIdentifiers ...string) {
+	logErrorInfo(err, callerSkip, infoIdentifiers...).Fatal(errorMsg)
+}
+
+func LogError(err error, errorMsg interface{}, callerSkip int, infoIdentifiers ...string) {
+	logErrorInfo(err, callerSkip, infoIdentifiers...).Error(errorMsg)
+}
 
-	pc, fullFilePath, line, ok := runtime.Caller(1)
+func logErrorInfo(err error, callerSkip int, infoIdentifiers ...string) *logrus.Entry {
```

Comment:
`infoIdentifiers` is not the best name. It does not identify info, the info will be simply named info_x (see below) ... this is the ACTUAL info itself, is it not?

Maybe
```suggestion
func logErrorInfo(err error, callerSkip int, additionalInfo ...string) *logrus.Entry {
```
?

Reply:
Sure, let's rename it. I would just use the plural **additionalInfos**.
It was called that in
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/96de9fe6db245b4064a268482967bea19b3535dd/handlers/common.go#L347
but I don't really know why.